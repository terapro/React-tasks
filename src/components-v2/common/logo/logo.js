import React from 'react';
import {Link} from 'react-router-dom';

export const Logo = () => (
  <Link to='/'>
    <img src='img/logo-netflixroulette.png' className='logo' />
  </Link>

);
