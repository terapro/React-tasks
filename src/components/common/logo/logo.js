import React from 'react';

const Logo =() => (
    <img src={"img/logo-netflixroulette.png"} className={'logo'} />
);

export {Logo};