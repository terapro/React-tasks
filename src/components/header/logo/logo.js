import React from 'react';

import {Logo} from '../common/logo/logo.js';

const Logo = () => (
  <div className={'header-logo'}>
    <Logo/>
  </div>

);

export {Logo};