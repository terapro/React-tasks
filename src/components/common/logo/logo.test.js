import React from 'react';
import {Logo} from 'src/components/common/logo/logo.js';
import {shallow} from 'enzyme';

describe('<Logo />', () => {
  it('should render the logo', () => {
    const wrapper = shallow(<Logo />);
    expect(wrapper).toMatchSnapshot();
  });
});