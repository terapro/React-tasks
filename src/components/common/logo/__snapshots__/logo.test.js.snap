// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should render the component 1`] = `
<img
  className="logo"
  src="img/logo-netflixroulette.png"
/>
`;
